README.md

```diff
@@ -1,2 +1,98 @@
+Interactive features, such as selecting, zooming, and panning, greatly enhance user experience.
+
+## Selection
+Select or highlight an area or shape to center the view over that area on a map.
+
+## Zooming and panning
+Zooming allows users to get a closer look at a particular area, and panning allows users to move a map around to focus on different areas.
+
+## Tooltips
+Tooltips display a pop-up with additional information about shapes, markers, and bubbles when the end user taps map elements.
+
+## Related links
+[Learn More about WinForms Maps](https://www.syncfusion.com/winforms-ui-controls/maps?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/windowsforms?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/products/windowsforms?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples)
+
+[Documentation](https://help.syncfusion.com/windowsforms/maps/getting-started?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples)
+
+[View Demos](https://github.com/syncfusion/winforms-demos/tree/master/maps?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/windowsforms?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/winforms?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples)
+
+[Samples](https://github.com/syncfusion/winforms-demos?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples)
+
+[Knowledge Base](https://www.syncfusion.com/kb/windowsforms?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples)
+
+## About Syncfusion WinForms UI Controls
+
+Syncfusion's [WinForms UI controls](https://www.syncfusion.com/winforms-ui-controls?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples) library is the only suite that you will ever need to build an application since it contains over 100 high-performance, lightweight, modular, and responsive UI Components in a single package. In addition to Diagram, we provide popular WinForms Controls such as [DataGrid](https://www.syncfusion.com/winforms-ui-controls/datagrid?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples), [Charts](https://www.syncfusion.com/winForms-ui-controls/chart?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples) [Scheduler](https://www.syncfusion.com/winforms-ui-controls/scheduler?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples),and [Syntax editor](https://www.syncfusion.com/winforms-ui-controls/syntax-editor?utm_source=github&utm_medium=listing&utm_campaign=winforms-maps-github-samples)
```

Comment:
Change to map
In addition to Map, we provide popular WinForms Controls......

Reply:
resolved